src/CMakeLists.txt

```diff
@@ -769,7 +769,7 @@ if (NOT WITH_SYSTEM_ROCKSDB)
   list(APPEND ROCKSDB_CMAKE_ARGS -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
 
   if (CMAKE_CXX_COMPILER_ID STREQUAL Clang)
-    list(APPEND ROCKSDB_CMAKE_ARGS -DFAIL_ON_WARNINGS=OFF)
+    list(APPEND ROCKSDB_CMAKE_ARGS -DFAIL_ON_WARNINGS=OFF -Wnoexpansion-to-defined)
```

Comment:
@tchaikov 
Compilation has been silenced. ;-) 
But I'm going to check if it was due to this change.